README.md

```diff
@@ -2,61 +2,109 @@
 
 We use the code in this repository to bootstrap our AWS PaaS environment. The normal flow is:
 
-1. Create the VPC under which the PaaS systems will live
-2. Create a Concourse using `bosh create-env`
-3. Deploy the pipelines that will spin up BOSH and CloudFoundry
-4. Sit back and wait
+1. Create the Infrastructure under which the Concourse, BOSH and PaaS systems will live
+2. Deploy a BOSH director
+3. Deploy CF, Concourse, Prometheus, etc.
+4. Deploy the automation pipelines for upgrades
 
 ## Pre-requisites
 
 - AWS CLI
 - Terraform CLI
+- BOSH CLI
+- CF CLI
+- CF Management CLIs
+- Credhub CLI
+- UAA CLI
+- PSQL client
+- MySQL client
 - Fly [CLI](https://concourse-ci.org/download.html)
 - [yq](https://github.com/mikefarah/yq) (or, `brew install yq`)
 - [jq](https://stedolan.github.io/jq/) (or, `brew install jq`)
 
 ## Creating a new environment
 
-You'll need to create a `<env>_vpc.tfvars` file with `az1`, `az2`, `region` and `parent_dns_zone`:
+1. Update submodules
 
-> **Note**: Multiple AZs are required in order to deploy an AWS ALB.
+```sh
+  make submodules
+```
 
-> set ingress_whitelist to the CIDRs that may access Concourse
+2. Create your environment directory and vars file
+
+```sh
+mkdir data
+touch data/<env>.tfvars
+```
+
+You'll need to create a `<env>.tfvars` file:
 
 ```json
 {
-"az1": "eu-west-1a",
-"az2": "eu-west-1b",
-"region": "eu-west-1",
-"parent_dns_zone": "<domain>",
-"ingress_whitelist": ["0.0.0.0/0"],
-"slack_webhook_uri": "https://hooks.slack.com/services/<generated uri>"
+    "environment": "<env>",
+    "region": "eu-west-1",
+    "availability_zones": ["eu-west-1a", "eu-west-1b", "eu-west-1c"],
+    "parent_dns_zone": "<parent domain>",
+    "ingress_whitelist": ["0.0.0.0/0"],
+    "vpc_cidr_block": "10.121.0.0/16",
+    "cidr_blocks": {
+        "public":     ["10.121.0.0/24", "10.121.1.0/24", "10.121.2.0/24"],
+        "internal":   ["10.121.8.0/22", "10.121.12.0/22", "10.121.16.0/22"],
+        "services":   ["10.121.28.0/22", "10.121.32.0/22", "10.121.36.0/22"],
+        "rds":        ["10.121.50.0/24", "10.121.51.0/24", "10.121.52.0/24"],
+        "prometheus": ["10.121.53.0/24", "10.121.54.0/24", "10.121.55.0/24"],
+        "concourse":  ["10.121.56.0/24", "10.121.57.0/24", "10.121.58.0/24"]
+    }
 }
 ```
 
-Example command:
+The `vpc_cidr_block` parameter is an array of IP ranges that you want to be able to access
+the PaaS. It should *not* be `0.0.0.0/0`.
+
+The `cidr_blocks` parameter specifies the IP subnet CIDR block ranges for each subnet and availability zone.
+This is specified to allow you to define exactly how big to make each subnet. (We could have automatically
+generated these values, but feel it is more comprehensible to view it in the base variables)
+
+3. Create an AWS user and access credentials
+
+You will need to create (manually) an AWS user and generate access and secret keys via the AWS console.
```

Comment:
Doesn't specify what permissions the user with the access key needs to have to be able to spin this up.